wordpress-mobile/WordPress-Android
# Dashboard Personalization: Remove today stats card remote feature flag

Parent #18944 

This PR removes the `my_site_dashboard_todays_stats_card` feature flag
The flag is no longer needed, as it was added over a year ago and we have fully committed to the today_stats card as part of the dashboard.
- Removed `MySiteDashboardTodaysStatsCardFeatureConfig`
- Always add `today_stats` to card types
- Updated unit tests

**To test:**
- Install the app
- Login to the app
- Select a site in which you have stats access
- Navigate to the My Site tab
- ✅ Verify the stats card shows in the dashboard
- Using the site selector, select a site in which you don't have stats access
- ✅ Verify the stats card is not shown in the dashboard


## Regression Notes
1. Potential unintended areas of impact
Stats card shows when it shouldn't

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing

3. What automated tests I added (or what prevented me from doing so)
CardsSourceTest 

PR submission checklist:

UI Changes testing checklist: N/A
